charts/tpu-dra-driver/values.yaml

```diff
@@ -0,0 +1,74 @@
+# Default values for draexample--driver.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+namespaceOverride: ""
+selectorLabelsOverride: {}
+
+allowDefaultNamespace: false
+
+deviceClasses: ["tpu"]
+
+imagePullSecrets: []
+image:
+  repository: us-central1-docker.pkg.dev/project-sri97/cloud-tpu
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+controller:
+  priorityClassName: "system-node-critical"
+  podAnnotations: {}
+  podSecurityContext: {}
+  nodeSelector: {}
+  tolerations:
+  - key: node-role.kubernetes.io/master
+    operator: Exists
+    effect: NoSchedule
+  - key: node-role.kubernetes.io/control-plane
+    operator: Exists
+    effect: NoSchedule
+  affinity: {}
+  containers:
+    controller:
+      securityContext: {}
+      resources: {}
+
+kubeletPlugin:
+  priorityClassName: "system-node-critical"
+  updateStrategy:
+    type: RollingUpdate
+  podAnnotations: {}
+  podSecurityContext: {}
+  nodeSelector: {}
+  tolerations: []
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: cloud.google.com/gke-tpu-accelerator
+            operator: Exists
+          - key: gke-no-default-tpu-device-plugin
```

Comment:
let's add `gke-no-default-tpu-dra-plugin` that doesn't exist now, but we will introduce it later so we can create built-in DRA driver and transition users of this helm chart to the built-in version